packages/block-editor/src/components/list-view/index.js

```diff
@@ -125,7 +131,7 @@ function ListView(
 	);
 
 	return (
-		<>
+		<AsyncModeProvider value={ true }>
```

Comment:
I suppose this could be `<AsyncModeProvider value>`, but that seems a bit unusual.